### PR TITLE
Change and document the architecture of the partials #49

### PR DESCRIPTION
**Pull request by aloscha**

### CODE REVIEW

File: src/Images/Api/MainHandlers.cs [hunk 1]
```diff
@@ -349,6 +368,10 @@ protected void RegisterMapperHandlers()
             UriMapping.OntologyMap("/images/partials/concept-chatattachment/{?}", typeof(ChatAttachment).FullName, (string objectId) => objectId, (string objectId) =>
             {
                 var illustration = DbHelper.FromID(DbHelper.Base64DecodeObjectID(objectId)) as Illustration;
+
+            UriMapping.OntologyMap("/images/partials/contents-edit/@w", typeof(EditAnnouncement).FullName);
```
Comment (warpech) [inline src/Images/Api/MainHandlers.cs]: This `OntologyMap`is declared before declaration of `OntologyMap` from line 368 is closed.

File: src/Images/Api/MainHandlers.cs [hunk 2]
```diff
@@ -276,6 +282,19 @@ protected void RegisterPartials()
                 }
             });
 
+            Handle.GET("/images/partials/image-mobile-edit/{?}", (string illustrationId) => {
```
Comment (aloscha) [inline src/Images/Api/MainHandlers.cs]: It should be removed, thanks

**Comment by warpech** [comment]
@aloscha any progress with this?

**Comment by aloscha** [comment]
I will provide changes tomorrow or on friday.

File: src/Images/Api/MainHandlers.cs (hunk 1 above)
Comment (aloscha) [inline src/Images/Api/MainHandlers.cs]: Damn, sorry. I have it in region and didn't see it.

**Comment by aloscha** [comment]
@warpech fixed

**Pull request merged**